lang/ar.js

```diff
@@ -54,6 +54,8 @@ export default {
 	"components.more-less.less": "أقل",
 	"components.more-less.more": "المزيد",
 	"components.overflow-group.moreActions": "مزيد من الإجراءات",
+	"components.scroll-wrapper.scrollBack": "scroll back",
+	"components.scroll-wrapper.scrollForward": "scroll forward",
```

Comment:
Tried to make these RTL-agnostic by going with "back" and "forward" instead of "scroll left" and "scroll right"